# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1204889119 18000
#      Fri Mar 07 06:25:19 2008 -0500
# Branch release-3-0-x
# Node ID c8b80a1af968aaea524b7315eea2c300316a4608
# Parent  c300c30e2ac6f0a702f328e27e95874fdd4159d6
set axes layer property to "top"

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-03-07  John W. Eaton  <jwe@octave.org>
+
+	* plot/contourf.m: Set axes layer property to "top".
+
 2008-03-06  John W. Eaton  <jwe@octave.org>
 
 	* plot/hist.m: Avoid temporaries.
 	Allow matrix arguments when number of bins > 30.
 	From Robert S. Mahurin <rob@utk.edu>.
 
 	* plot/__go_draw_axes__.m: Adjust markersize by a factor of 1/6.
 
diff --git a/scripts/plot/contourf.m b/scripts/plot/contourf.m
--- a/scripts/plot/contourf.m
+++ b/scripts/plot/contourf.m
@@ -175,16 +175,18 @@ function varargout = contourf (varargin)
   endfor
 
   if (min (lev) == max (lev))
     set (gca (), "clim", [min(lev)-1, max(lev)+1]);
   else
     set (gca(), "clim", [min(lev), max(lev)]);
   endif
 
+  set (gca (), "layer", "top");
+
   if (nargout > 0)
     varargout{2} = h;
     varargout{1} = c;
   endif
 
 endfunction
 
 function [X, Y, Z, lvl, patch_props] = parse_args (arg)
